### PR TITLE
chore: update dependencies

### LINKED ISSUE #69: Update to Angular 13
Update dependencies to latest

### PR DESCRIPTION
**Pull request by TapaiBalazs**
## Chore

Updated angular
fixed compilation issues

Closes #69

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,61 +42,65 @@
...
+    "@angular/cdk": "13.1.1",
+    "@angular/common": "13.1.1",
+    "@angular/compiler": "13.1.1",
+    "@angular/core": "13.1.1",
+    "@angular/forms": "13.1.1",
+    "@angular/material": "13.1.1",
+    "@angular/platform-browser": "13.1.1",
+    "@angular/platform-browser-dynamic": "13.1.1",
+    "@angular/router": "13.1.1",
+    "@nrwl/angular": "13.4.3",
```
Comment (luixaviles) [inline package.json:55]: I may suggest using either `~version` or `^version` to not miss future minor/patch versions on dependencies.

**Review by luixaviles — APPROVED** [review]
Just left a suggestion for managing dependencies on `package.json` file. Everything else looks good to me.

File: package.json (hunk 1 above)
Comment (TapaiBalazs) [reply package.json:55]: that is entirely handled by NX. I'd don't want to change a thing with these.

Comment (TapaiBalazs) [reply package.json]: I added `^` to these

**Review by ktrz — APPROVED** [review]
LGTM

**Pull request merged**